src/docs/forms/components/CheckBox/examples/01-SimpleCheckBox.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import Form, {
+    CheckBox,
+    TextInput
+} from 'doj-react-adminlte/Form';
+
+export default class SimpleCheckBox extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            checkbox1: null,
+            checkbox2: null,
+            checkbox3: null,
+            checkbox4: null
+        };
+    }
+
+    handleChange = (name, value) => {
+        this.setState({
+            [name]: value
+        });
+    };
+
+    render() {
+        const {checkbox1,checkbox2,checkbox3,checkbox4} = this.state;
+        return (
+            <div>
+                <Form onChange={this.handleChange}>
+                    <div className="row">
+                        <div className="col-xs-12">
+                            <span className="text-bold">School Subjects</span>
+                            <CheckBox label="MATH"
```

Comment:
Change example to list other items instead. (e.g., fruits, vegetables, names of persons, etc.). Please use sentence case.